native/packages/react-native-bpk-component-image/README.md

```diff
@@ -66,16 +66,50 @@ export default () => (
 );
 ```
 
+### With A Custom Image Component
+
+```js
+import React, { Component } from 'react';
+import { StyleSheet, View } from 'react-native';
+import { spacingBase, spacingLg } from 'bpk-tokens/tokens/base.react.native';
+import BpkImage from 'react-native-bpk-component-image';
+import CustomImage from './CustomImage';
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    justifyContent: 'center',
+    padding: spacingBase,
+  },
+  image: {
+    width: '100%',
+    height: spacingLg * 10,
+  },
+});
+
+export default () => (
+  <View style={styles.container}>
+    <BpkImage
+      imageComponent={CustomImage}
```

Comment:
We usually don't include optional props in the readme example.
If we do, it tends to be a separate example